src/test/java/teammates/ui/webapi/DeleteDeadlineExtensionActionTest.java

```diff
@@ -0,0 +1,69 @@
+package teammates.ui.webapi;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.DeadlineExtensionAttributes;
+import teammates.common.util.Const;
+import teammates.ui.output.MessageOutput;
+
+/**
+ * SUT: {@link DeleteAccountRequestAction}.
+ */
+public class DeleteDeadlineExtensionActionTest extends BaseActionTest<DeleteDeadlineExtensionAction> {
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.DEADLINE_EXTENSION;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return DELETE;
+    }
+
+    @Override
+    @Test
+    protected void testExecute() {
+        DeadlineExtensionAttributes deadlineExtension = typicalBundle.deadlineExtensions.get("student4InCourse1Session1");
```

Comment:
Has this been added to the "db" and is ensured that it exists prior to delete action execution?

Reply:
Will add a check to verify existence of deadline of deadline extension for good measure.